widget/discrepancy-note/dn-widget.js

```diff
@@ -574,12 +574,10 @@ class Comment extends Widget {
             widget.style[ o[ 0 ] ] = o[ 1 ];
         } );
 
-        // Scroll directly to Query modal if not enough height for user to view the Query modal
-        // https://github.com/OpenClinica/enketo-express-oc/issues/378
-        if ( this.linkedQuestion.classList.contains( 'or-appearance-image-map' ) ) {
-            setTimeout( () => {
-                this._scrollToview();
-            }, 500 );
+        // https://github.com/OpenClinica/enketo-express-oc/issues/495
+        const maps = document.querySelectorAll( '.or-appearance-image-map' );
```

Comment:
Each question gets its own widget. You are going through all the image-maps in the whole form for each image map dn question, so there is a lot of duplication if multiple image-map questions exist in the form.

Reply:
So this means you should query `this.linkedQuestion` instead of `document`

Reply:
This is still outstanding (just in case it was overlooked).